plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v2.test.ts

```diff
@@ -33,6 +33,8 @@ import { AttributeNames } from '../src/enums';
 import { mockV2AwsSend } from './testing-utils';
 import { expect } from 'expect';
 import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
+//just to match the name, even if it changes. if path change this will crash
+import { ExceptionEventName } from '@opentelemetry/sdk-trace-base/build/src/enums';
```

Comment:
looks like other places in the repo just use the constant string 'exception', examples: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-js-contrib%20%27exception%27&type=code

I would recommend doing the same here too so that we don't import from an internal file as you mentioned.